dev/io.openliberty.microprofile.telemetry.internal/bnd.bnd

```diff
@@ -16,7 +16,9 @@ Bundle-SymbolicName: io.openliberty.microprofile.telemetry.internal; singleton:=
 
```

Comment:
I hope I am not contradicting myself here but shouldn't this bundle be called `io.openliberty.io.opentelemetry` and then placed in the dev folder?
`publish.wlp.jar.suffix: dev/api/stable`

Reply:
However, the jar that goes in the dev folder should only contain the external APIs ... the ones you list as APIs in the public feature. The rest should be separated out.

Reply:
The naming of the API bundle is hard because there aren't really any MicroProfile APIs, they're all io.opentelemetry. How about using `io.openliberty.io.opentelemetry` (APIs in the dev folder) and `io.openliberty.io.opentelemetry.internal` (the rest of the impl in the lib folder)

Reply:
or merge the rest of the impl packages into `io.openliberty.microprofile.telemetry.1.0.internal`